skin-schema.json

```diff
@@ -1226,7 +1276,7 @@
     "buttonNames": {
       "description": "These are the only buttons currently supported by the UI SDK.",
       "additionalProperties": false,
-      "enum": ["playPause", "volume", "timeDuration", "flexibleSpace", "discovery", "fullscreen", "stereoscopic",  "moreOptions", "arrowsBlack", "logo", "share", "closedCaption", "bitrateSelector", "quality", "live", "multiAudio"]
```

Comment:
Is `bitrateSelector` no longer needed?

Reply:
I tried to find bitrateSelector in 
- html5-skin
- tml5-video-plugins
- html5-ad-plugins
- html5-analytics-plugins
- video-plugins-internal
- html5-common
- mjolnir

And bitrateSelector doesn not exist there. 
I found bitrateSelector in https://github.com/ooyala/native-skin but it is for sdk.

Reply:
@KseniiaSokolovaEpam the skin config is shared between web and sdks, so we shouldn't remove anything that's used by the sdks.

Reply:
Oh, ok, I got it. Thank you! I will return it back.

Reply:
\+